test/ghosts.tests.js

```diff
@@ -28,7 +29,8 @@ test('ghosts.add, ghosts.get, ghosts.getMinDepth', async (t) => {
     })
     msgIDs.push(rec.id)
   }
-  const tangleID = peer.db.feed.findMoot(account, 'post')?.id
+  const moot = MsgV4.createMoot(account, 'post', keypair)
+  const tangleID = MsgV4.getMsgID(moot)
```

Comment:
had to do this to get it to pass because `tangleID` was null/undefined for some reason (i had promisified it too). but the findMoot tests were all passing so :shrug:

Reply:
Maybe `findMoot` needs to be async too? It doesn't sound right that after `feed.publish`ing posts (after which the moot should definitely be in the db), `findMoot` would return null.

Generally, if something isn't working as expected, then there's a problem there which shouldn't be ignored.

Reply:
true, was being lazy. there were some obvious cases i think i might've missed, switching back now will work if i'm lucky

Reply:
ah lmao i think i must've added an `await` to the promisified fn but it was trying to await the `?.id` not the function. so added parens around the await now and it works